db/migrate/20170116200958_self_referential_orchestration_template.rb

```diff
@@ -0,0 +1,8 @@
+class SelfReferentialOrchestrationTemplate < ActiveRecord::Migration[5.0]
+  def change
+    change_table :orchestration_templates do |t|
+      t.references :parent, index: true, comment: "Nested template reference"
+      t.string :url, comment: "URL for nested/child template"
```

Comment:
Yeah, it's a thing https://www.postgresql.org/docs/9.5/static/sql-comment.html

I haven't really used it a ton, but I could see it being useful in some cases. Plus the meanings of columns don't change very often.

On the down side, the comment is not very visible to the developers and I don't think many rails devs would go to `psql` looking for information about particular columns. Does rails display the column comment in some way where it would be visible to the developers or is this just at the database level?